pkgs/by-name/fl/flake-edit/package.nix

```diff
@@ -46,6 +45,8 @@ rustPlatform.buildRustPackage rec {
     installShellCompletion --zsh --name _flake-edit target/assets/_flake-edit
   '';
 
+  passthru.updateScript = nix-update-script { };
```

Comment:
Maybe want to ignore 0.1.0 as the update script would find it as being newer.

Reply:
The previously released version is `0.0.1`, but it was packaged as `0.1.0`, the script would do everything correctly.